deploy/charts/alluxio/templates/helpers/_alluxio-env.tpl

```diff
@@ -62,8 +62,8 @@ See the NOTICE file distributed with this work for information regarding copyrig
 {{- end -}}
 
 {{- define "alluxio.env" -}}
-ALLUXIO_MASTER_JAVA_OPTS+="{{ include "alluxio.master.env" . }}"
-ALLUXIO_WORKER_JAVA_OPTS+="{{ include "alluxio.worker.env" . }}"
-ALLUXIO_PROXY_JAVA_OPTS+="{{ include "alluxio.proxy.env" . }}"
-ALLUXIO_FUSE_JAVA_OPTS+="{{ include "alluxio.fuse.env" . }}"
+ALLUXIO_MASTER_JAVA_OPTS="{{ include "alluxio.master.env" . }}"
```

Comment:
https://github.com/Alluxio/alluxio/blob/c75317feb8843133e0e9e3211c06a9871d0eb1cc/cli/src/alluxio.org/cli/env/env.go#L99C23-L99C23
This method only parses `=` but not `+=`